### PR TITLE
MohammadH/Refactor Testimonial

### PR DESCRIPTION
**Pull request by mohammad-rework**
Changes:

-   Refactor Testimonial

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — CHANGES_REQUESTED** [review]

File: src/pages/home/_what-our-clients-say.tsx [hunk 1]
```diff
@@ -105,267 +105,127 @@ const testimonial_slides = [
     {
         id: 'angeh',
         name: 'Angeh',
-        quote: (
-            <Localize translate_text="Weekend trades, fast deposits & withdrawals, plus synthetics trading - what’s better than this?" />
-        ),
```
[the remark this answers is not in the document]
Comment (mohammad-rework) [reply src/pages/home/_what-our-clients-say.tsx:110]: I used Localize in line 228. Doesn't it work?

**Review by prince-deriv — DISMISSED** [review]

File: src/pages/home/_what-our-clients-say.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sean-binary) [reply src/pages/home/_what-our-clients-say.tsx:110]: Not for the string extraction. As that is basically a node script that scans through all files and extract anything wrapped with `localize`.

[the remark this answers is not in the document]
Comment (sean-binary) [reply src/pages/home/_what-our-clients-say.tsx:110]: You can refer to this for more info: https://deriv-group.slack.com/archives/G3AHMSM3P/p1575604609012800

**Review by vijayasree-deriv — APPROVED** [review]

**Review by alamin-deriv — APPROVED** [review]
LGTM

**Review by sean-binary — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**